config.go

```diff
@@ -129,7 +129,7 @@ func newDefaultConfig() appConfig {
 
 type Config struct {
 	appsConfig map[string]*appConfig
-	file       *os.File
+	filepath   string
```

Comment:
It's fine here.

In general, it's good to have pascal or camel case for variables and fields where package names are all lowercase such that there is no way of conflict.